### PR TITLE
Update with my stats

### PR DESCRIPTION
**Pull request by blauks**

### CODE REVIEW

**Review by no-ran — COMMENTED** [review]
Thank you for your pr!!
But maybe you want to change line 5 too?

**Comment by blauks** [comment]
Of course, good spot!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ### Hey Hey People
 ## I code sometimes:
-[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=no-ran&show_icons=true&theme=vision-friendly-dark)](https://github.com/anuraghazra/github-readme-stats)
+[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=blauks&show_icons=true&theme=vision-friendly-dark)](https://github.com/anuraghazra/github-readme-stats)
 <br />
 ![GitHub stats](https://github-readme-stats.vercel.app/api?username=no-ran&show_icons=true&theme=vision-friendly-dark)
```
Comment (blauks) [inline README.md]:
```suggestion
![GitHub stats](https://github-readme-stats.vercel.app/api?username=blauks&show_icons=true&theme=vision-friendly-dark)
```

**Review by no-ran — APPROVED** [review]
After carefully reviewing your PR, I must say I have never seen better code, Well Done!!

**Comment by blauks** [comment]
Great, thanks! I can always count on you

**Pull request merged**